lib/run.js

```diff
@@ -134,9 +134,9 @@ function _run (opts, cb, tracker) {
     totalCompletedRequests += counter
     requests.recordValue(counter)
     throughput.recordValue(bytes)
+    tracker.emit('tick', counter)
```

Comment:
What are your thoughts on making this line like this? 
```suggestion
    tracker.emit('tick', {counter, bytes})
```

If you think users would want access to the `count` per tick, would it be fair to think they'd want `bytes` per tick, too?

Reply:
Great suggestion. I just updated the event. :)